resources/default-config.kdl

```diff
@@ -16,6 +16,13 @@ input {
         // Delay is in milliseconds before the repeat starts. Rate is in characters per second.
         // repeat-delay 600
         // repeat-rate 25
+
+        // How compositor should track the layout.
+        //
+        // - "global" - layout change is global for all windows.
+        // - "window" - layout is tracked for each window individually.
+        //
+        //track-layout "global"
```

Comment:
I think this would make more sense as a boolean toggle:
```suggestion
        // Uncomment this line to make the layout change per-window, rather than globally.
        // per-window-layout
```

Reply:
I'm not sure about boolean, the enum is at lest clear what it does by default.